.pre-commit-config.yaml

```diff
@@ -1,27 +1,50 @@
+default_stages: [pre-commit, pre-push]
```

Comment:
Removing it will make all of the current `pre-commit` hooks to run on push as well though I believe - https://pre-commit.com/#top_level-default_stages. I can test it out tomorrow.

Reply:
```suggestion
default_stages: [pre-commit]
```
I wasn't talking about removing this, I was talking about making the default `pre-commit` as suggested above, and so you won't need to specify the stages except for `pre-push` hooks.

Reply:
Ah I understand now